packages/fast-check/test/unit/arbitrary/_internals/StreamArbitrary.spec.ts

```diff
@@ -160,13 +175,13 @@ describe('StreamArbitrary', () => {
           stringify.mockImplementation(fakeStringify);
 
           // Act
-          const arb = new StreamArbitrary(sourceArb);
+          const arb = new StreamArbitrary(sourceArb, true);
           const stream = arb.generate(mrng, biasFactor).value;
           const values = [...stream.take(expectedValues.length)];
 
           // Assert
           expect(values).toEqual(expectedValues);
-          expect(String(stream)).toEqual(`Stream(${expectedValues.map(fakeStringify).join(',')}…)`);
```

Comment:
We could have kept the toEqual check given we preserved the stringified value for the history mode

Reply:
Fixed.